### PR TITLE
XRT-2209 feat(opc-ua-server): Added Modbus tcp, bacnet ip and virtual services to opc-ua server examples

### PR DESCRIPTION
**Pull request by Tylerg2471**

### CODE REVIEW

File: Servers/opc-ua/basic/README.md [hunk 1]
```diff
@@ -30,6 +46,9 @@ We have provided a script to easily set these environment variables. Run:
 cd Servers/opc-ua/basic/
 . ../../../set_env_vars.sh
 export S7_SIM_ADDRESS=0.0.0.0
+export BACNET_IP_SIM_ADDRESS=<ip_of_sim>
+export MODBUS_SIM_ADDRESS=$<ip_of_sim>
```
Comment (BinduRao2018) [inline Servers/opc-ua/basic/README.md]: use the same convention : should be export MODBUS_SIM_ADDRESS=<ip_of_sim>  (without $)

File: set_env_vars.sh [hunk 2]
```diff
@@ -8,9 +8,19 @@ export XRT_MQTT_USERNAME=""
 export XRT_MQTT_PASSWORD=""
 
 # Just used for ApplicationComponent Example
-export IOT_DIR=/opt/iotech/iot/1.4
+export IOT_DIR=/opt/iotech/iot/1.5
 export XRT_DIR=/opt/iotech/xrt/2.1
```
Comment (BinduRao2018) [inline set_env_vars.sh]: XRT_DIR - change it to 2.2

**Review by BinduRao2018 — CHANGES_REQUESTED** [review]
Minor comments in-line

File: Servers/opc-ua/basic/README.md (hunk 1 above)
Comment (Tylerg2471) [reply Servers/opc-ua/basic/README.md]: Fixed

File: set_env_vars.sh (hunk 2 above)
Comment (Tylerg2471) [reply set_env_vars.sh]: Updated

**Review by BinduRao2018 — APPROVED** [review]
LGTM

**Pull request merged**